.travis.yml

```diff
@@ -134,53 +134,44 @@ jobs:
         - STAGE=deploy
         - NAME=deploy
       script:
-        # Update tag for version defined in lfs.h
+        # Find version defined in lfs.h
         - LFS_VERSION=$(grep -ox '#define LFS_VERSION .*' lfs.h | cut -d ' ' -f3)
         - LFS_VERSION_MAJOR=$((0xffff & ($LFS_VERSION >> 16)))
         - LFS_VERSION_MINOR=$((0xffff & ($LFS_VERSION >>  0)))
-        - LFS_VERSION="v$LFS_VERSION_MAJOR.$LFS_VERSION_MINOR"
-        - echo "littlefs version $LFS_VERSION"
+        # Grab latests patch from repo tags, default to 0
+        - LFS_VERSION_PATCH=$(curl -f -u "$GEKY_BOT_RELEASES"
```

Comment:
Is it a security concern, if someone opens a PR on this repo with a modified .travis.yml, which does an `echo $GEKY_BOT_RELEASES`, they would be able to do anything that geky-bot can do?

Reply:
Good question, GEKY_BOT_RELEASES is stored in a secure variable in Travis ([more info](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings)). This offers a few protections. Most notably, secure variables are not available on PRs, only on "trusted" builds. So you can't actually get GEKY_BOT_RELEASES on PRs.

Also, I (unintentionally) found out that Travis filters out any secure variables if they get printed accidentally. You could still figure out the variables with a sort of dictionary attack, but still, that would have to land on master.

And finally, the actual value of GEKY_BOT_RELEASES is a GitHub token of the form "geky-bot:12341234123412341234123412341234". So it's easy to disable if it does get compromised.

As a sidenote, this is why GEKY_BOT_STATUSES and GEKY_BOT_RELEASES are two different variables. GEKY_BOT_STATUSES is a separate GitHub token that only has permissions to update commit statuses and is available on PRs for reporting CI results.

Reply:
Ok, thanks for the explanation, that's useful to know!